backend/typescript/prisma/schema.prisma

```diff
@@ -5,7 +5,7 @@ generator client {
 
 datasource db {
   provider      = "postgresql"
-  url           = env("DATABASE_URL")
+  url           = "postgresql://postgres:postgres@localhost:5432/scv2"
```

Comment:
Interesting, so what happened here? Was prisma not pulling the database url as expected from the env file?

Reply:
Yes, there were some issues accessing the db. I accidentally pushed this change, though — I had meant to keep it locally. I'll revert this change.

If other people also experience issues (specifically regarding localhost:5432 not being accessible), we found that changing the database url to `postgresql://postgres:postgres@de_db:5432/scv2` in the .env worked. However, the url in the `schema.prisma` must then be changed to `postgresql://postgres:postgres@localhost:5432/scv2`.